modules/OCPP/manifest.yaml

```diff
@@ -77,10 +77,10 @@ requires:
     interface: evse_manager
     min_connections: 1
     max_connections: 128
-  connector_zero_sink:
+  evse_energy_sink:
     interface: external_energy_limits
     min_connections: 0
-    max_connections: 1
+    max_connections: 129
```

Comment:
why is the 129 here and 128 in the OCPP201 module manifest?

Reply:
129 is correct. 
1 for EVSE id 0
128 for the evse manager modules